docs-source/content/userguide/managing-domains/accessing-the-domain/admin-console.md

```diff
@@ -1,6 +1,25 @@
 ---
-title: "Administration Console"
+title: "Use the Remote Console"
 date: 2019-02-23T17:39:15-05:00
-draft: true
+draft: false
 weight: 2
+description: "Use the Oracle WebLogic Server Remote Console to manage a domain running in Kubernetes."
 ---
+Use the Remote Console to access WebLogic domains running in Kubernetes.
+
+The Oracle WebLogic Server Remote Console is a lightweight, open source console that does not need to be collocated with a WebLogic Server domain.
+You can install and run the Remote Console anywhere. For an introduction, read the blog, "The NEW WebLogic Server Remote Console."
+For detailed documentation, see the [Oracle WebLogic Server Remote Console](https://github.com/oracle/weblogic-remote-console) GitHub project.
+
+A major benefit of using the Remote Console is that you don't need to install or run the WebLogic Server Administration Console on WebLogic Server instances.
```

Comment:
Two other things you might want to mention:

- A second benefit is that when accessing the Remote Console through a load balancer, it's easier to setup access to multiple different domains than the WLS Administration Console.  

- A drawback is that the Remote Console does not currently display monitoring information (this is coming in a future release).

But I'm not sure, @rjeberhard and @mriccell may have a different opinion.

Reply:
Thank you. @tbarnes-us This doc is a work in progress. Your suggestions will be taken into consideration.

Reply:
I do not want to document drawbacks to the Remote Console in the Operator project, limitations are already documented in the Remote Console project.

Reply:
> A major benefit of using the Remote Console is that you don't need to install or run the WebLogic Server Administration Console on WebLogic Server instances.

Checking with you all... when I read this sentence, I hear it as, "Now you can use WebLogic less!" I don't think that this is what you mean :). How about something like, "A major benefit of using the Remote Console is that it runs in your browser and can be used to connect to different WebLogic Server instances."

Reply:
@mriccell @rjeberhard Tom's suggestion is ok with me. But, I think that Monica's intent is to emphasize that you can use the WLS slim installer and reduce the size of the download, etc. Opinions?

Reply:
I like the idea of emphasizing that the customer can use the slim installer. I just objected to the wording that made it seem like you wouldn't be running WebLogic at all. I do like Tom's "second" benefit of the remote console as it is complicated to expose either the traditional console or the JMX port through a load balancer.

Reply:
okey doke. I'll work it in.